Datatable/Column/AbstractColumn.php

```diff
@@ -752,13 +760,14 @@ public function getDql()
      * Set dql.
      *
      * @param null|string $dql
+     * @param bool        $validate Validates DQL.
      *
      * @return $this
      * @throws Exception
      */
-    public function setDql($dql)
+    public function setDql($dql, $validate = false)
     {
-        if (true === $this->dqlConstraint($dql)) {
+        if ((false == $validate) || (true === $this->dqlConstraint($dql))) {
             $this->dql = $dql;
```

Comment:
call setDql('foo', true)
possibly: If validate = true, the result of the "dqlConstraint" function is irrelevant